src/tests/functional/plugin/shared/include/behavior/ov_executable_network/get_metric.hpp

```diff
@@ -345,6 +346,29 @@ TEST_P(OVClassHeteroExecutableNetworkGetMetricTest_TARGET_FALLBACK, GetMetricNoT
     ASSERT_EQ(expectedTargets, targets);
 }
 
+TEST_P(OVClassHeteroExecutableNetworkGetMetricTest_EXEC_DEVICES, GetMetricNoThrow) {
+    ov::Core ie = createCoreWithTemplate();
+    std::vector<std::string> expectedTargets = {target_device};
+#ifdef ENABLE_INTEL_CPU
```

Comment:
we must not have plugin dependent test in common part. Could you please rework the tests to be sustainable and not plugin dependent?

Reply:
@ilya-lavrenov will work on this. @wgzintel please noted on this requirement as well